pandas/tests/window/test_rolling.py

```diff
@@ -1170,7 +1171,10 @@ def test_rolling_sem(frame_or_series):
     tm.assert_series_equal(result, expected)
 
 
-@pytest.mark.xfail(is_platform_arm() and not is_platform_mac(), reason="GH 38921")
+@pytest.mark.xfail(
```

Comment:
I guess we have to figure out what's wrong in here at some point

Reply:
Yeah definitely. Maybe we should reconsider if it's also worth using a sliding window algorithm if it's susceptible to these numerical precision issues

Reply:
Hm I think this is a compiler issue. Something is optimised that triggers the problem.

Reply:
Right a compiler flag seems to match the consensus in issue ( https://github.com/pandas-dev/pandas/issues/38921 ).

Namely it looks like some optimization causes a form of [`fma`]( https://en.cppreference.com/w/c/numeric/math/fma ) to get called, which evaluates incorrectly. This implementation in Cython seems to demonstrate the issue ( https://github.com/pandas-dev/pandas/issues/38921#issuecomment-800884529 ).

Am guessing this issue would also show up on macOS ARM (even though that hasn't come up yet).

Reply:
Agreed. At least for this PR I am just hoping to skip this test for now until a fix is investigated

Reply:
Yep my comment was meant more generally.

yes this also happens on Mac, already validated this